api/db/migrations/arch_carpool/20230920000000-create_carpool_v2.up.sql

```diff
@@ -0,0 +1,158 @@
+CREATE SCHEMA IF NOT EXISTS carpool_v2;
+
+CREATE TABLE IF NOT EXISTS carpool_v2.carpools
+(
+  _id SERIAL PRIMARY KEY,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  operator_id INTEGER NOT NULL REFERENCES operator.operators(_id),
+  operator_journey_id VARCHAR NOT NULL,
+  operator_trip_id VARCHAR,
+  operator_class CHAR NOT NULL,
+  start_datetime TIMESTAMP WITH TIME ZONE NOT NULL,
+  start_position GEOGRAPHY NOT NULL,
+  end_datetime TIMESTAMP WITH TIME ZONE NOT NULL,
+  end_position GEOGRAPHY NOT NULL,
+  distance INT NOT NULL,
+  licence_plate VARCHAR(32),
+
+  driver_identity_key VARCHAR(64),
+  driver_operator_user_id VARCHAR(64),
+  driver_phone VARCHAR(32),
+  driver_phone_trunc VARCHAR(32),
+  driver_travelpass_name VARCHAR(32),
+  driver_travelpass_user_id VARCHAR(128),
+  driver_revenue INTEGER NOT NULL,
+
+  passenger_identity_key VARCHAR(64),
+  passenger_operator_user_id VARCHAR(64),
+  passenger_phone VARCHAR(32),
+  passenger_phone_trunc VARCHAR(32),
+  passenger_travelpass_name VARCHAR(32),
+  passenger_travelpass_user_id VARCHAR(128),
+  passenger_over_18 BOOLEAN,
+  passenger_seats SMALLINT NOT NULL,
+  passenger_contribution INTEGER NOT NULL,
+  passenger_payments JSONB
+);
+
```

Comment:
On a pas d'index `UNIQUE` ?

Reply:
J'ai crée celui-là. C'est ça dont tu parles ?

CREATE UNIQUE INDEX IF NOT EXISTS carpool_operator_operator_journey_id_idx ON carpool_v2.status (operator_id, operator_journey_id);